src/main/java/com/paneedah/mwc/weapons/Magazines.java

```diff
@@ -22,7 +22,7 @@ public class Magazines {
     public static ItemMagazine SOCOM_Mag;
     public static ItemMagazine HK417Mag;
     public static ItemMagazine C8Mag;
-    public static ItemMagazine SA80Mag;
+//    public static ItemMagazine SA80Mag;
```

Comment:
It is preferable to just delete this, anyway thanks to GitHub if we ever need it we can just see it in the history.